game/scripts/npc/items/item_bloodthorn.txt

```diff
@@ -65,14 +65,14 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityValues"
     {
-      "bonus_intellect"                                   "20 30 45 65 90"
+      "bonus_intellect"                                   "20 25 30 35 40"
       "bonus_attack_speed"                                "60 70 80 90 100"
       "bonus_damage"                                      "50 60 80 110 150"
       "bonus_mana_regen"                                  "5 5.25 5.75 6.5 7.5"
       "bonus_magic_resist"                                "20 21 22 23 24"
       "silence_duration"                                  "5"
       "silence_damage_percent"                            "30"
-      "target_crit_multiplier"                            "130 135 145 160 180"
+      "target_crit_multiplier"                            "130 140 150 160 170"
```

Comment:
I am not sure if this is a good idea. This is going back to the values when Bloodthorn was S tier item.

Reply:
I changed it for the greatest of all reasons, it looks really stupid.